adafruit_ssd1306/ssd1306.py

```diff
@@ -38,72 +46,95 @@ def __init__(self, width, height, external_vcc):
         self.init_display()
 
     def init_display(self):
+        """base class initialize display"""
         for cmd in (
-            SET_DISP | 0x00, # off
-            # address setting
-            SET_MEM_ADDR, 0x00, # horizontal
-            # resolution and layout
-            SET_DISP_START_LINE | 0x00,
-            SET_SEG_REMAP | 0x01, # column addr 127 mapped to SEG0
-            SET_MUX_RATIO, self.height - 1,
-            SET_COM_OUT_DIR | 0x08, # scan from COM[N] to COM0
-            SET_DISP_OFFSET, 0x00,
-            SET_COM_PIN_CFG, 0x02 if self.height == 32 else 0x12,
-            # timing and driving scheme
-            SET_DISP_CLK_DIV, 0x80,
-            SET_PRECHARGE, 0x22 if self.external_vcc else 0xf1,
-            SET_VCOM_DESEL, 0x30, # 0.83*Vcc
-            # display
-            SET_CONTRAST, 0xff, # maximum
-            SET_ENTIRE_ON, # output follows RAM contents
-            SET_NORM_INV, # not inverted
-            # charge pump
-            SET_CHARGE_PUMP, 0x10 if self.external_vcc else 0x14,
-            SET_DISP | 0x01): # on
+                SET_DISP | 0x00, # off
+                # address setting
+                SET_MEM_ADDR, 0x00, # horizontal
+                # resolution and layout
+                SET_DISP_START_LINE | 0x00,
+                SET_SEG_REMAP | 0x01, # column addr 127 mapped to SEG0
+                SET_MUX_RATIO, self.height - 1,
+                SET_COM_OUT_DIR | 0x08, # scan from COM[N] to COM0
+                SET_DISP_OFFSET, 0x00,
+                SET_COM_PIN_CFG, 0x02 if self.height == 32 else 0x12,
+                # timing and driving scheme
+                SET_DISP_CLK_DIV, 0x80,
+                SET_PRECHARGE, 0x22 if self.external_vcc else 0xf1,
+                SET_VCOM_DESEL, 0x30, # 0.83*Vcc
+                # display
+                SET_CONTRAST, 0xff, # maximum
+                SET_ENTIRE_ON, # output follows RAM contents
+                SET_NORM_INV, # not inverted
+                # charge pump
+                SET_CHARGE_PUMP, 0x10 if self.external_vcc else 0x14,
+                SET_DISP | 0x01): # on
             self.write_cmd(cmd)
         self.fill(0)
         self.show()
 
     def poweroff(self):
+        """poweroff"""
```

Comment:
Please be a bit more verbose here and start the comments with a capital letter. (Applies here and elsewhere.)